src/Validators/validateNumberInRange.ts

```diff
@@ -0,0 +1,36 @@
+import { validateNumber } from "../Validators";
+import { NumberIsTooLargeError, NumberIsTooSmallError } from "../Errors";
+
+const validateNumberGreaterThan = (min: IRangeBound, aNumber: number) => {
+  if (min.value === undefined) return;
+  const isValid = min.include ? aNumber >= min.value : aNumber > min.value;
+  if (isValid) return;
+  throw new NumberIsTooSmallError(min.value, min.include);
+};
+
+const validateNumberSmallerThan = (max: IRangeBound, aNumber: number) => {
+  if (max.value === undefined) return;
+  const isValid = max.include ? aNumber <= max.value : aNumber < max.value;
```

Comment:
siento que el include, not include deberian ser 2 metodos pero bueno, no me molesta tanto

Reply:
no entiendo, es un parametro, como lo haces un metodo?